### PR TITLE
added filter for function type and a uniq filter

### PR DESCRIPTION
**Pull request by twilson63**
For stamps, added a couple of additional filters:

* to only return stamp interactions
* and to make sure they are uniq, no duplicates

> In theory, someone could post a bunch of invalid transactions and create some fake Stamps with this query.

> You may want to look into calling https://cache.permapages.app/{STAMP CONTRACT} and filter the stamps object, happy to help if you want to consider this approach.

### CODE REVIEW

File: src/permapages/img/stamps/api.js [hunk 1]
```diff
@@ -9,19 +10,23 @@ export async function getStamps(address) {
...
+          stampedAsset: inputTag?.transactionId,
+          stamper: stamp["tags"].find((tag) => tag.name === "Sequencer-Owner")
+            ?.value,
+          timestamp: inputTag?.timestamp,
+        });
+      }
     }
 
     if (address) {
       return feed.filter((stamp) => stamp.stamper === address);
```
Comment (charmful0x) [inline src/permapages/img/stamps/api.js]: Can you please, inside this code block replace the `feed.filter()` with your `uniqBy` addition? Also to keep a return `feed` function incase no address was supplied (`address` is optional)

File: src/permapages/img/stamps/api.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import { makeQueries } from "./gql.js";
+//import uniqBy from 'lodash.uniqby'
```
Comment (charmful0x) [inline src/permapages/img/stamps/api.js]: Please uncomment this import :)

**Review by charmful0x — CHANGES_REQUESTED** [review]
I left comments asking for minor changes, thank you for the PR! :))

File: src/permapages/img/stamps/api.js (hunk 1 above)
Comment (twilson63) [reply src/permapages/img/stamps/api.js]: Yep

File: src/permapages/img/stamps/api.js (hunk 2 above)
Comment (twilson63) [reply src/permapages/img/stamps/api.js]: doh!

**Review by charmful0x — APPROVED** [review]

**Pull request merged**